roles/contiv_network/tasks/main.yml

```diff
@@ -35,6 +35,9 @@
 - name: copy systemd units for netplugin
   copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
 
+- name: copy bash auto complete file for netctl
+  shell: sudo cp /usr/bin/contrib/completion/bash/netctl /etc/bash_completion.d/ && sudo rm -rf /usr/bin/contrib
```

Comment:
this is just minor:

consider using the `file` module and make this a symlink to the extracted file. It will keep all the netplugin related extracted files in one location (say `usr/bin/netplugin/`), than scattered at multiple locations with symlink placed in desired locations. I find this a bit cleaner  (easier to debug like it easier to find if a file really comes from the last extracted release etc) but totally up to you to incorporate.